ansible/ansible.cfg

```diff
@@ -1,8 +1,9 @@
 [defaults]
-callback_whitelist = profile_tasks
+callback_whitelist = profile_tasks,datadog_callback
 host_key_checking = False
-inventory = inventory/openstack.yml
+inventory = inventory
 timeout = 30
+vault_password_file = vault-env
```

Comment:
I assume that for some Ansible-specific general rule `vault_password_file` var here is equivalent to `--vault-password-file` from CLI.

Based on https://docs.ansible.com/ansible/2.4/vault.html#providing-vault-passwords
* The recommended options is `--vault-id` (rather than `--vault-password-file`);
* I see `vault-env` is a "a vault password executable script" though this capability of being a script, whose stdout is relevant, rather than a plain file, whose content is relevant, is documented for `--vault-id` - not for `--vault-password-file`.

Reply:
Well, ansible.cfg options are documented, while it may looks like there 1:1 match with the cli options, I don't think it's the case.

I see only [`vault_password_file` documented](docs.ansible.com/ansible/2.4/intro_configuration.html#vault-password-file) and no `vault_id` or related.

Reply:
Ah great thanks. And documented to be potentially a script. Thanks.